lib/advisor/web/controllers/login_controller.ex

```diff
@@ -1,25 +1,26 @@
 defmodule Advisor.Web.LoginController do
   use Advisor.Web, :controller
   alias Advisor.Core.People
+  alias Advisor.Web.Authentication.Password
 
-  def index(conn, %{"email" => email, "submit" => "dashboard"}) do
-    login(conn, as: email, redirect_to: "/dashboard")
+  def index(conn, %{"email" => email, "password" => pw, "submit" => "dashboard"}) do
```

Comment:
Wouldn't start with `"submit" =>` make it easier to catch the difference between the methods?